### ISSUE TITLE
Bad zoom behaviour when switching back to app [during split-view on iPad]

### ISSUE THREAD

**Issue by onionmad** [home-assistant/iOS#2109]
**iOS device model, version and app version**

Model Name: iPad Pro 2018, 11"
Software Version: Latest stable
App version: Latest stable

**Home Assistant Core Version**

**Describe the bug**
When closing (minimizing) and returning to the app, as the app returns, the zoom levels are (very often) increased without user input. This only occurs during split-view(side-by-side) in landscape device orientation.  Tihs also occurs if 'pinch to zoom' is disabled in the app settings, with the only difference being that there is no way to zoom back out then!

**To Reproduce**
Set one or two instances of the companion app in split view, using landscape device orientation, and alt-tab (if using keyboard) or simply multi-task out of the split view, opening another app and then returning to the split view (companion apps), one or both companion app instances will be zoomed in from the default full view. 

**Expected behavior**
Being able to multi-task in and out of the companion app without any zoom adjustments. 

**Screenshots**

**Additional context**

Does not occur in normal single instance-full-screen mode. Only during split-view with the companion app as one or both visible apps.

**Comment by jazzmonger** [comment]
I'm glad someone else reported this.  It is absolutely maddening when this happens.  
ios version 17.1.2, but is happened ever since I installed it years ago. It's the ONLY app that does this.

On my iPad it happens whenever I task switch over to companion after a few minutes.
Then I get a screen that looks like this

![IMG_2630](https://github.com/home-assistant/iOS/assets/52110065/f018b12e-5fb6-47a6-859a-6f8ee9338879)

it seems to be happening about 50% of the time these days and has gotten worse over time. I

**Comment by Stooovie** [comment]
Unfixed as of 2024.6

**Comment by onionmad** [comment]
FYI, I have worked around this issue by simply disabling the 'pinch to zoom' option. At least I'm fairly sure that's the setting that fixed it...

**Comment by Stooovie** [comment]
No, on the contrary - when I disable that option, I'm unable to zoom out and have to force quit the app.
